HealthcareSystem/GraphicalEditorServerTests/DataFactory/CreateEquipmentType.cs

```diff
@@ -5,21 +5,16 @@
 
 namespace GraphicalEditorServerTests.DataFactory
 {
-    public class CreateEquipmentType : ICreateTestObjectFactory<EquipmentType>
+    public class CreateEquipmentType 
     {
         public EquipmentType CreateInvalidTestObject()
         {
-            return new EquipmentType("fsdafafas",false);
+            return null;
         }
```

Comment:
I'd probably delete this method entirely instead of returning null.

Reply:
Alright, thanks for advice